is-the-average-whole/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <script>
+    myArray1 = [3, 4]
+    myArray2 = [1, 3, 4, 4]
+    myArray3 = [5, 5, 5]
+    function isTheAverageWhole(numbersArray){
+        var sum = 0;
+        for (var i = 0; i < numbersArray.length; i++) {
+          sum += numbersArray[i];
+        }
+        var avg = sum / numbersArray.length;
+        var a = avg.toString()
+        var periodt = '.';
+        var check = a.includes(periodt)
+        console.log(numbersArray, !check)
```

Comment:
Return a value, **Do not console log it**

The purpose of these practice problems is to create a reusable function

Think of a real world scenario where you have a data set of numbers and you need to check if the average of these numbers is a whole number

```js
var myData = [2, 3, 4 ,3, 42]
var isDataWhole = isTheAverageWhole(myData)
if(isDataWhole) {
// true logic
} else {
// false logic
}
```

`console.log` in production is reserved for debugging.  Use return statements.

Reply:
I updated it, sorry I forgot again!!